packages/integrations/cloudflare/README.md

```diff
@@ -159,6 +159,25 @@ export function GET({ params }) {
 }
 ```
 
+### Astro Dev Server
+
+The Adapter allows an opt-in flag, to enable the use of Environment Variables and the Cloudflare Request Object to be populated by the Astro Dev Server. There is no need to use wrangler in this case. This flag is `off` by default.
+
+There are two supported modes: `local` & `remote`. In the mode `local` environmental variables are available, but the request object is populated from a static placeholder value. In the mode `remote` environmental variables and the a live fetched request object are available.
```

Comment:
```suggestion
### `cloudflare.runtime` 

`runtime: "off" | "local" | "remote"`
default `"off"`

This optional flag enables the Astro dev server to populate environment variables and the Cloudflare Request Object, avoiding the need for Wrangler.

-  `local`: environmental variables are available, but the request object is populated from a static placeholder value. 
- `remote`: environmental variables and the live, fetched request object are available.
- `off`: the Astro dev server will populate neither environment variables nor the request object. Use Wrangler to access Cloudflare bindings and environment variables.
```

I don't think "dev server" is the key idea here, it's the config object. I took a stab at this from the structure of the `mode` section above. Please make this true/in the right jargon, but I think this kind of structure will work well here!